packages/experimental/Avatar/src/__tests__/JSAvatar.test.jsx

```diff
@@ -18,7 +18,7 @@ const alphabeticalTestNames = [
   'Richard Feynman {Teams}',
 ];
 const alphabeticalInitials = ['M', 'AK', 'MB', 'AK', 'AK', 'WL', 'WL', 'AK', 'RF', 'R', 'RF', 'RF'];
-const otherTestNames = ['(206) 123-4567', '耿盈盈']; // will return this case after bug bash - '耿 James'
+const otherTestNames = ['(206) 123-4567', '耿盈盈', '47 12 34']; // will return these cases after bug bash - '1x1', '耿 James', '+47 12 34 56 78 (X 5678)'
```

Comment:
we should remove any references to the bug bash in the code

Reply:
I'll remove these cases for now because maybe we won't need this method at all when I add language support